src/rpc/blockchain.cpp

```diff
@@ -331,6 +333,7 @@ UniValue getblockheader(const UniValue& params, bool fHelp)
             "  \"previousblockhash\" : \"hash\",  (string) The hash of the previous block\n"
             "  \"nextblockhash\" : \"hash\",      (string) The hash of the next block\n"
             "  \"chainwork\" : \"0000...1f3\"     (string) Expected number of hashes required to produce the current chain (in hex)\n"
+            "  \"nextblockhash\" : \"hash\"       (string) The hash of the next block\n"
```

Comment:
maybe rebase error?  nextblockhash is repeated.  to keep the order the same as output, chainwork should go before previousblockhash and nextblockhash.